app/routes.py

```diff
@@ -0,0 +1,413 @@
+from app import db
```

Comment:
This file is over 400 lines of code! We can refactor the overall project structure by adding the logic specific to the customer, video, and rental routes into separate files within a `routes` folder.